frontend/src/components/RolePermissionsCard.js

```diff
@@ -0,0 +1,229 @@
+                  <Checkbox
+                    checked={state.permissions.admin}
+                    onChange={handleChange}
+                    name="admin"
+                  />
+                }
+                label="Admin"
+              />
+              <FormControlLabel
+                className={classes.indentedCheckbox}
+                disabled={!state.permissions.admin}
+                control={
+                  <Checkbox
+                    checked={state.permissions.edit_project}
+                    onChange={handleChange}
+                    name="edit_project"
+                  />
+                }
+                label="Edit Existing Project"
+              />
+              <FormControlLabel
+                className={classes.indentedCheckbox}
+                disabled={!state.permissions.admin}
+                control={
+                  <Checkbox
+                    checked={state.permissions.create_project}
+                    onChange={handleChange}
+                    name="create_project"
+                  />
+                }
+                label="Create New Project"
+              />
+              <FormControlLabel
+                className={classes.indentedCheckbox}
+                disabled={!state.permissions.admin}
+                control={
+                  <Checkbox
+                    checked={state.permissions.manage_role}
+                    onChange={handleChange}
+                    name="manage_role"
+                  />
+                }
+                label="Manage Role"
+              />
+              <FormControlLabel
+                className={classes.indentedCheckbox}
+                disabled={!state.permissions.admin}
+                control={
+                  <Checkbox
+                    checked={state.permissions.activate_accounts}
+                    onChange={handleChange}
+                    name="activate_accounts"
+                  />
+                }
+                label="Activate/Deactivate Accounts"
+              />
+            </FormGroup>
+          </FormControl>
+
+          {mode === "edit" ? (
+            <Button
+              className={`${classes.deleteButton} ${classes.warning}`}
+              variant="contained"
+              startIcon={<DeleteIcon />}
+              onClick={(event) => {
+                event.preventDefault();
+
+                // TODO - Implement delete functionality
+              }}
+            >
+              Delete Role
+            </Button>
+          ) : (
+            ""
+          )}
+        </CardContent>
+      </Card>
+
+      <Grid container xs={12} alignItems="center" justify="center">
+        <Button
+          className={classes.saveButton}
+          variant="contained"
+          color="primary"
+          onClick={(event) => {
+            event.preventDefault();
+            // TODO - Implement save function
+          }}
+        >
+          Save Changes
```

Comment:
This button needs to say Save Changes or Create depending on if we came from editing or creating